ui/packages/consul-ui/app/components/consul/upstream/list/index.hbs

```diff
@@ -24,15 +24,22 @@ as |item index|>
         </Tooltip>
       </dd>
     </dl>
-    {{#if (and (env 'CONSUL_NSPACES_ENABLED') (not-eq item.Namespace @nspace))}}
-      <a data-test-service-name href={{href-to 'nspace.dc.services.show' (concat '~' item.Namespace) @dc item.Name }}>
+{{#if (and (env 'CONSUL_NSPACES_ENABLED') (not-eq item.Namespace @nspace))}}
```

Comment:
Does this need to use `can` now?

Reply:
We decided we would make a new ability called `canSee()`.

Reply:
I know I said `canSee` yesterday and I know this is quibbling over words, but I was just about to do this and figured that it's not about whether you can see them or not, its about whether you can use them.. ha I just thought its about whether you have them or not, but maybe `canHaz()` `can('haz nspaces')` isn't the best 😆 

Think we can go with canAccess or canUse or something similar 🤔

Reply:
Oh look I just saw this:

https://github.com/hashicorp/consul/blob/ce958e72180d80fb8da054a9483a8172f83076ce/ui/packages/consul-ui/app/abilities/acl.js#L22-L24

I'll go with `canUse` then 👍